src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -259,40 +255,31 @@ class TutorialDirectoryUi extends React.Component {
   };
 
   renderHeader = () => {
-    const notices = this.renderNotices();
     const headerLinks = this.renderHeaderLinks();
-
     return (
-      <>
-        <EuiFlexGroup alignItems="center">
-          <EuiFlexItem>
-            <EuiTitle size="l">
-              <h1>
-                <FormattedMessage
-                  id="home.tutorial.addDataToKibanaTitle"
-                  defaultMessage="Add data"
-                />
-              </h1>
-            </EuiTitle>
-          </EuiFlexItem>
-          {headerLinks ? <EuiFlexItem grow={false}>{headerLinks}</EuiFlexItem> : null}
-        </EuiFlexGroup>
-        {notices}
-      </>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiTitle size="l">
+            <h1>
+              <FormattedMessage id="home.tutorial.addDataToKibanaTitle" defaultMessage="Add data" />
+            </h1>
+          </EuiTitle>
+        </EuiFlexItem>
+        {headerLinks ? <EuiFlexItem grow={false}>{headerLinks}</EuiFlexItem> : null}
+      </EuiFlexGroup>
     );
   };
 
   render() {
     return (
-      <EuiPage restrictWidth={1200}>
-        <EuiPageBody>
-          {this.renderHeader()}
-          <EuiSpacer size="m" />
-          <EuiTabs>{this.renderTabs()}</EuiTabs>
-          <EuiSpacer />
-          {this.renderTabContent()}
-        </EuiPageBody>
-      </EuiPage>
+      <KibanaPageTemplate restrictWidth={1200}>
```

Comment:
Can you also move the page header implementation to the `pageHeader` prop here?
```suggestion
      <KibanaPageTemplate 
        restrictWidth={1200}
        pageHeader={{
          pageTitle: (<FormattedMessage />),
          rightSideItems: [headerLinks],
        }}
      >
```